library/manageiq.py

```diff
@@ -97,9 +97,9 @@ def update_provider(self, provider_id, provider_name, endpoints):
         """ Updates the existing provider with new parameters
         """
         try:
-            result = self.client.post(self.providers_url + '/%d' % provider_id,
-                                      action='edit',
-                                      connection_configurations=endpoints)
+            self.client.post(self.providers_url + '/%d' % provider_id,
```

Comment:
@simon3z https://github.com/dkorn/manageiq-ansible-module/pull/5 handles the string formatting you suggested